README.md

```diff
@@ -12,3 +12,4 @@ The images in here are used in the following situations:
 | gui-e2e-testing | gui-e2e-testing | [gui](https://github.com/mendersoftware/gui/blob/master/tests/e2e_tests/docker-compose.e2e-tests.yml) |
 | backend-integration-testing | backend-integration-testing | [integration](https://github.com/mendersoftware/integration/blob/master/backend-tests/docker/docker-compose.backend-tests.yml) for use in [Mender QA](https://github.com/mendersoftware/mender-qa/blob/master/gitlab-pipeline/stage/test.yml) |
 | mender-client-acceptance-testing | mender-client-acceptance-testing | [Mender QA](https://github.com/mendersoftware/mender-qa/blob/master/.gitlab-ci.yml) |
+| mender-dist-packages-builder-{DISTRO}-{RELEASE}-{ARCH} | mender-dist-packages-building | [mender-dist-packages](https://github.com/mendersoftware/mender-dist-packages/blob/master/.gitlab-ci.yml) |
```

Comment:
but according to the other PR we are not bulding a separate image, but we use a tag. see other comment in the corresponding PR.

Reply:
But this row in the table lists the name of the tag, not the image. See the other comment in the corresponding PR :smiley: